File: javaprogram/Maximum.java
import java.lang.*;
import java.util.*;
class Maximum {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);
    System.out.print("How many numbers do you want to read to find maximum number, enter the number: ");
    int Limit = scanner.nextInt();
    System.out.print("Enter the first number: ");
    int Numbers[] = new int[Limit];
    Numbers[0] = scanner.nextInt();
    int Count = 1;
    while(Count  < Limit) {
      System.out.print("Enter the next number: ");
      Numbers[Count] = scanner.nextInt();
      Count = Count + 1;
    }
    Count = 1;
    int MaximumNumber = Numbers[0];
    while(Count < Limit) {
      if(MaximumNumber < Numbers[Count]) {
        MaximumNumber = Numbers[Count];
      }
      Count = Count + 1;
    }
    Count = 1;
    System.out.print("The maximum number of " + Numbers[0]);
    while(Count < Limit) {
      System.out.print(", "+Numbers[Count]);
      Count = Count + 1;
    }
    System.out.print(" is " + MaximumNumber + ".");
    System.out.println("");
  }
}

File: Maximum.c
#include <stdio.h>

int main(void) {
  int Number[100], Count = 0, MaximumNumber = 0, Limit = 0;
  printf("To find the maximum number n numbers, enter the limit:");
  scanf("%d", &Limit);
  printf("Enter the first number: ");
  while(Count < Limit) {
    scanf("%d", &Number[Count];
    Count = Count + 1;
    if(Count < Limit) {
      printf("Enter the next number: ");
    }
  }
  Count = 0;
  while(Count < Limit) {
    if(MaximumNumber < Number[Count]) {
      MaximumNumber = Number[Count];
    }
    Count = Count + 1; 
  }
  Count = 1;
  printf("The maximum number of %d",Number[0]);
  while(Count < Limit) {
    printf(", %d",Number[Count]);
    Count = Count + 1;
  }
  printf(" is %d.", MaximumNumber);
  printf("\n");
  return 0;
}